src/Symfony/Contracts/HttpClient/Test/Fixtures/web/index.php

```diff
@@ -29,7 +29,7 @@
     }
 }
 
-$json = json_encode($vars, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
+$json = json_encode($vars, \JSON_PRETTY_PRINT | \JSON_UNESCAPED_SLASHES | \JSON_UNESCAPED_UNICODE);
```

Comment:
This code runs in the global namespace already. Why would we want backslashes here?

Reply:
reverted